src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/Options/OptionsExtensions.cs

```diff
@@ -97,10 +115,23 @@ private static void MapObject(object obj, string prefix, IDictionary<string, str
             {
                 if (!property.GetIndexParameters().Any())
                 {
-                    MapValue(
-                        property.GetValue(obj),
-                        FormattableString.Invariant($"{prefix}{property.Name}"),
-                        map);
+                    if (property.GetCustomAttribute<System.Text.Json.Serialization.JsonExtensionDataAttribute>() != null)
```

Comment:
I don't think this change is necessary anymore due to the egress configuration refactor.